ontology/yaml/resources/HVAC/entity_types/AHU.yaml

```diff
@@ -3737,164 +3731,302 @@ AHU_CHWSC_ECON_FDPM_EDRPC_RFSS_RFVSC_SFSS_SFVSC_SSPC:
   - SFVSC
   - SSPC
 
-###################################
-### Existing Non-standard Types ###
-###################################
-
-# TODO: Remove pressure points from AHU_US_MTV_1250_16
-# TODO: Remove zone air co2 setpoint from AHU_US_MTV_1300_2
-# TODO: Every AHU must have outside_air_temperature_sensor, so almost every AHU is currently incomplete.
-#       See b/148117417 for details
-
-AHU_US_LVK_7633_1: # This building isnt found in the BMS anymore... Where is it?
-  id: "5764350924508102656"
-  description: "Non-standard type for 7633"
+AHU_DFSS_DX2ZC_ECONZ_CO2C_DSP_RCKTM2X_HTZC_CC2XM_RTM_DTM:
+  id: "2060882401344618496"
+  description: "AHU with dual compressor refrigeration circuits (with monitoring), heating, and zone-level temperature and ventilation control."
+  is_canonical: true
   implements:
-  - REMAP_REQUIRED
-  uses:
-  - building_air_static_pressure_sensor
-  - building_air_static_pressure_setpoint
-  - compressor_run_command_1
-  - compressor_run_command_2
-  - compressor_run_command_3
-  - compressor_run_command_4
-  - compressor_speed_percentage_command
-  - economizer_mode
-  - exhaust_air_damper_percentage_command
-  - exhaust_fan_run_command
-  - exhaust_fan_run_status
-  - exhaust_fan_speed_frequency_sensor
-  - exhaust_fan_speed_percentage_command
-  - mixed_air_temperature_sensor
-  - outside_air_damper_percentage_command
-  - outside_air_flowrate_sensor
-  - outside_air_flowrate_setpoint
-  - return_air_temperature_sensor
-  - supply_air_static_pressure_sensor
-  - supply_air_static_pressure_setpoint
-  - supply_air_temperature_sensor
-  - supply_air_temperature_setpoint
-  - supply_fan_run_command
-  - supply_fan_run_status
-  - supply_fan_speed_frequency_sensor
-  - supply_fan_speed_percentage_command
-  - supply_air_flowrate_sensor
+  - AHU
+  - DFSS
+  - DX2ZC
+  - ECONZ
+  - CO2C
+  - DSP
+  - RCKTM2X
+  - HTZC
+  - CC2XM
+  - RTM
+  - DTM
+  opt_uses:
+  - condensing_fan_current_sensor
 
-AHU_US_LVK_7633_2: # This building isnt found in the BMS anymore... Where is it?
-  id: "6039492714242768896"
-  description: "Non-standard type for 7633"
+AHU_SFSS_SFMSC_SFVSC_FDPM_CHWSC_RTM_RACO2C_SSPC_VOADM2X_SAIDC_RAIDC2X:
+  id: "5588410372366270464"
```

Comment:
How did you get the `id`? it is usually generated by Phred?

Reply:
the ids were already generated. i changed the names for certain types, but they will continue to have the same IDs.

Reply:
@charbull ping reminder to approve PR :)